ACKategories-iOS/SwiftUIExtensions/FontModifier.swift

```diff
@@ -0,0 +1,34 @@
+import SwiftUI
+
+@available(iOSApplicationExtension 14.0, *)
```

Comment:
I sort of think we should go with `@available(iOS 14.0, *)` I think that is the reason why the Cocoapods check failed 🤔